src/PureCloud.Client/Apis/WebChatApi.cs

```diff
@@ -265,53 +293,51 @@ public async Task<WebChatDeployment> UpdateWebchatDeploymentAsync(string deploym
 
         var client = _httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
 
-        var uri = UriHelper.GetUri($"api/v2/webchat/deployments/{deploymentId}", null);
-
-        var response = await client.PutAsJsonAsync(uri, body, _options.JsonSerializerOptions, cancellationToken);
+        var response = await client.PutAsJsonAsync($"api/v2/webchat/deployments/{Uri.EscapeDataString(deploymentId)}", body, _options.JsonSerializerOptions, cancellationToken);
 
         response.EnsureSuccessStatusCode();
 
         return await response.Content.ReadFromJsonAsync<WebChatDeployment>(_options.JsonSerializerOptions, cancellationToken);
     }
 
     /// <inheritdoc />
-    public async Task DeleteWebchatDeploymentAsync(string deploymentId, CancellationToken cancellationToken = default)
+    public async Task<bool> DeleteWebchatDeploymentAsync(string deploymentId, CancellationToken cancellationToken = default)
     {
         ArgumentException.ThrowIfNullOrEmpty(deploymentId);
 
         var client = _httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
 
-        var uri = UriHelper.GetUri($"api/v2/webchat/deployments/{deploymentId}", null);
-
-        var response = await client.DeleteAsync(uri, cancellationToken);
+        var response = await client.DeleteAsync($"api/v2/webchat/deployments/{Uri.EscapeDataString(deploymentId)}", cancellationToken);
 
         response.EnsureSuccessStatusCode();
```

Comment:
Return IsSucceasStatusCode instead of this. Apply this everywhere we return Boolean